.github/codecov.yml

```diff
@@ -40,6 +40,55 @@ flag_management:
         target: auto # Let's decrease this later.
         threshold: 10
 
+  groups:
+    gno-land:
+      paths:
+        - "gno.land/**"
+      flags:
+        - gno-land
+    gnovm:
+      paths:
+        - "gnovm/**"
+      flags:
+        - gnovm
+    misc:
+      paths:
+        - "misc/**"
+      flags:
+        - misc
+    tm2:
+      paths:
+        - "tm2/**"
+      flags:
+        - tm2
+    contribs:
+      paths:
+        - "contribs/**"
+      flags:
+        - contribs
+
+  statuses:
+    - type: project
+      flag: gno-land
+      target: auto
+      threshold: 5
+    - type: project
+      flag: gnovm
+      target: auto
+      threshold: 2 # Stricter threshold
+    - type: project
+      flag: misc
+      target: auto
+      threshold: 10
+    - type: project
+      flag: tm2
+      target: auto
+      threshold: 5
+    - type: project
+      flag: contribs
+      target: auto
+      threshold: 5
```

Comment:
why do we want to change thresholds depending on the flag? Is it really necessary?

Reply:
It's to give us finer grained control over each specific sub-project (flag)